src/adagrams.js

```diff
@@ -1,8 +1,134 @@
-const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
-};
+class Adagrams  {
+  static drawLetters() {
+      const AllLetters = ['a','a','a','a','a','a','a','a','a','b','b','c','c','d','d','d','d','e','e','e','e','e','e','e','e','e','e','e','e','f','f','g','g','g','h','h','i','i','i','i','i','i','i','i','i','j','k','l','l','l','l','m','m','n','n','n','n','n','n','o','o','o','o','o','o','o','o','p','p','q','r','r','r','r','r','r','s','s','s','s','t','t','t','t','t','t','u','u','u','u','v','v','w','w','x','y','y','z'];
+      const handy = [];
+      for (let i = 1; i < 11; i++) {
+        let index = [Math.floor(Math.random()*AllLetters.length)];
+        let removed = AllLetters.splice(index, 1);
+        let removed2 = removed.pop()
+        handy.push(removed2.toUpperCase());
+      }
+      return handy;
+  }
+  static usesAvailableLetters (input, lettersInHand) {
+    let value = true;
```

Comment:
I would probably prefer a different variable name, as `value` doesn't really mean anything to me in terms of understanding the code